dp/cloud/python/magma/configuration_controller/response_processor/strategies/response_processing.py

```diff
@@ -337,6 +327,19 @@ def _create_grant_from_response(
     return grant
 
 
+def _remove_grant_from_response(
+        response: DBResponse, session: Session, unset_freq: bool = False,
+) -> None:
+    grant = _get_grant_from_response(response, session)
+    if not grant:
+        return
+
+    if unset_freq:
+        unset_frequency(grant)
+    session.delete(grant)
+    logger.info('Terminated grant')
```

Comment:
Please add at least grantId to the logger for tracking purposes and log debugging.